lib/routes/drivers/tests/routes.test.js

```diff
@@ -0,0 +1,167 @@
+import { resetAndMockDB } from 'utils/testUtils';
+
+const payload = {
+    firstName: 'Jhon',
+    lastName: 'Wick',
+    email: 'wick@jhon.is',
+    mobileNo: '8585959524',
+    cabTypeId: 1,
+    cabNumber: 'MH 20 9211',
+    gender: 'male'
+};
```

Comment:
A usual convention we have to move this to `mockData.js` we also stick with SCREAMING_SNAKE_CASE for constants.